packages/mobile/src/screens/sign-on-screen/screens/SelectArtistScreen/SelectArtistsScreen.tsx

```diff
@@ -45,8 +58,13 @@ export const SelectArtistsScreen = () => {
   )
 
   const handleSubmit = useCallback(() => {
-    toast({ content: 'TODO: Create Account' })
-  }, [toast])
+    if (accountCreationStatus === EditingStatus.LOADING) {
+      navigation.navigate('AccountLoading')
+    } else {
+      dispatch(finishSignUp())
```

Comment:
double checking the "welcome to audius" drawer opens?

Reply:
oh actually good catch, the modal logic wasn't there before so I need to add it 👍